stream/stream_bluray.c

```diff
@@ -482,7 +474,7 @@ static int bluray_stream_open_internal(stream_t *s)
     // initialize libbluray event queue
     bd_get_event(bd, NULL);
 
-    select_initial_title(s, title_guess);
+    select_initial_title(s, bd_get_main_title(bd));
```

Comment:
The `meson.build` should be bumped to 0.5.0 apparently.
[https://code.videolan.org/videolan/libbluray/-/blob/master/ChangeLog#L254](https://code.videolan.org/videolan/libbluray/-/blob/master/ChangeLog#L254)

Reply:
Sure, I checked only if it's available in old enough lib.